microprofile-health/src/test/java/org/jboss/eap/qe/microprofile/health/MultiDeploymentHealthTest.java

```diff
@@ -0,0 +1,106 @@
+package org.jboss.eap.qe.microprofile.health;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.hasItems;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.as.arquillian.api.ContainerResource;
+import org.jboss.as.arquillian.container.ManagementClient;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.EmptyAsset;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+
+/**
+ * Multiple deployment scenario.
+ */
+@RunWith(Arquillian.class)
+public class MultiDeploymentHealthTest {
+
+    @ContainerResource
+    ManagementClient managementClient;
+
+    String healthURL;
+
+    @Before
+    public void composeHealthEndpointURL() {
+        healthURL = "http://" + managementClient.getMgmtAddress() + ":" + managementClient.getMgmtPort() + "/health";
+    }
+
+    @Deployment(name = "deployment1", order = 1, testable = false)
+    public static Archive<?> deployment1() {
+        WebArchive war = ShrinkWrap.create(WebArchive.class, MultiDeploymentHealthTest.class.getSimpleName() + "-1.war")
+                .addClasses(SimplifiedHealthCheck.class)
+                .addAsWebInfResource(EmptyAsset.INSTANCE, "beans.xml");
+        return war;
+    }
+
+    @Deployment(name = "deployment2", order = 2, testable = false)
+    public static Archive<?> deployment2() {
+        WebArchive war = ShrinkWrap.create(WebArchive.class, MultiDeploymentHealthTest.class.getSimpleName() + "-2.war")
+                .addClasses(LivenessHealthCheck.class)
+                .addAsWebInfResource(EmptyAsset.INSTANCE, "beans.xml");
+        return war;
+    }
+
+    /**
+     * @tpTestDetails Customer multi-deployment scenario.
+     * @tpPassCrit Overall and the health check status is up.
+     * @tpSince EAP 7.4.0.CD19
+     */
+    @Test
+    @RunAsClient
+    public void testHealthEndpoint() {
+        RestAssured.when().get(this.healthURL).then()
+                .contentType(ContentType.JSON)
+                .header("Content-Type", containsString("application/json"))
+                .body("status", is("UP"),
+                        "checks", hasSize(3), // BothHealthCheck contains both annotations: @Liveness and @Readiness
```

Comment:
Here again I think it's `BothHealthCheck` -> `SimplifiedHealthCheck` or just remove the comment.

Reply:
Fixing...